src/pages/patterns/global-header-pattern/index.mdx

```diff
@@ -0,0 +1,299 @@
+
+<Row>
+
+<Column  colLg={12} colMd={12} colSm={6} >
+
+![hint text](images/nav_priority_changes.png)
+
+<Caption>
+  1. Navigation items appear in the header at wide widths. <Br />
+  2. Navigation items have moved to a side nav for narrow widths.
+</Caption>
+
+</Column>
+</Row>
+
+#### Navigating content by headers
+
+Users who rely on screen reader technology to navigate areas a site also rely on screen readers to navigate the content of a page. In the same way users can visually scan for the larger and bolder type of a header, users of assistive technologies also must understand the hierarchy of the page's content to efficiently navigate a page's content.
+
+To ensure all users interpret the structure of your content in the same way, the visual representation of a heading should match the underlying ranking of the header tag. Tutorials on how to achieve this can be found in the WCAG page structure [tutorials on headings](https://www.w3.org/WAI/tutorials/page-structure/headings/).
+
+### Keyboard navigation
+
+#### Skip to main content
+
+Some users may use a keyboard to navigate your site. Starting focus in the main nav lets them quickly navigate to other areas in your UI, but could block them from the main content if there is a large number of nav items to tab through first.
+
+[Success Criterion 2.4.1 (Bypass Blocks)](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G1) suggest bypassing these blocks by providing "Skip to main" link at the start of you navigation's focusable controls. This lets users easily skip the navigation region and being interacting with the page's main content area.
+
+<Row>
+<Column  colLg={8} colMd={8} colSm={6} >
+
+![hint text](images/skip-to-main-content.gif)
+
+<Caption>
+  The "Skip to main content link" is the first focusable element on the Carbon
+  website.
+</Caption>
+
+</Column>
+</Row>
+
+#### Landmark regions
+
+Landmark regions are a way of grouping similar areas of content in your UI and assigning them roles. This technique lets users navigating with assistive technologies quickly get around your site by navigating between landmark regions in your UI rather than each individual element.
+
+Examples of common landmark regions include: navigation, main, form, banner, and search. If there are multiple navigation landmark regions give each a unique label.
+
+<Row>
+
+<Column  colLg={8} colMd={8} colSm={6} >
+
+![hint text](images/landmark_zones.png)
+
+<Caption>
+  1. Navigation landmark. <Br />
+  2. Search landmark. <Br />
+  3. Main landmark. <Br />
+</Caption>
+
+</Column>
+
+</Row>
+
+#### Navigating the page via regions
+
+Navigating between landmark regions helps users who cannot see the visual grouping of your navigation. This grouping can be important to understanding the organization of the structure of the content and making it clear what users can do and where they can go in your UI.
+
+<Row>
+
+<Column  colLg={8} colMd={8} colSm={6} >
+
+![hint text](images/voiceover.png)
```

Comment:
For preview this is fine, but we need to add detailed alt text to each of these images before merging.